docs/V4_API_Updates.md

```diff
@@ -45,6 +45,14 @@ Purchases.configure(
 )
 ```
 
+### New Types
```

Comment:
Not the most informative but we can expand this before release.

Reply:
[Made a task for it](https://app.shortcut.com/revenuecat/story/12569/detailed-documentation-of-usage-and-motivation-of-new-types-that-wrap-storekit-types)